parallel_walk_test.go

```diff
@@ -2,30 +2,28 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
-package goftp_test
```

Comment:
Can we change this back to _test package?

Reply:
I believe I left the *tests* (as opposed to the *examples*) as `package goftp` so it’s easier to fork this package to a different GitHub path (as I did) and send pull requests.

I believe changing the examples to `package goftp_test` met your feedback requirement that the example have the package name in the example code. I think it’s a great pattern that we’ll adopt in our other public Go packages!

Reply:
Done in 531cdf9